dataset/dictionary/y-events.json5

```
@@ -2001,7 +2092,10 @@
     en: "Reminiscent Regimen",
     ja: "追憶練行",
     zhCN: "追想练行",
-    notes: "v2.8 期間限定イベント",
+    pronunciationJa: "ついおくれんぎょう",
+    notesEn: "Limited Time Event in v2.8",
+    notes: " v2.8 期間限定イベント",
```

I guess you accidentally inserted a leading space.


```suggestion
    notes: "v2.8 期間限定イベント",
```